tasks/main.yml

```diff
@@ -45,6 +45,15 @@
     - ansible_distribution == "Debian"
     - ansible_distribution_release == "stretch"
 
+# Amazon Linux 2023 does not (yet?) offer docker-compose or the
+# Docker compose plugin:
+# https://github.com/amazonlinux/amazon-linux-2023/issues/186
+- name: Install docker compose plugin V2 (Amazon Linux)
+  ansible.builtin.include_tasks:
+    file: compose_Amazon.yml
+  when:
+    - ansible_distribution == "Amazon"
```

Comment:
Do we want this to cover all Amazon Linux distros or just AL2023?  If just AL2023, we should be specific here.

Reply:
While I have not officially included testing for the legacy Amazon Linux distros in this PR, this task would also apply to prior distros and therefore the more generalized `ansible_distribution` still seems appropriate here.

Reply:
I think @sei-dspooner is correct.  We are clear about which versions of Amazon Linux are supported in the role metadata (i.e., only 2023), so a check against `amazon_distribution` is sufficient here.  This also agrees with what we have historically done when Amazon required special treatment.

@sei-dspooner, in the future please let the reporter be the one to resolve conversations, since he or she is the person best able to judge if you have sufficiently addressed the issue that was brought up.

Reply:
This works for me, thanks for the clarifications. 👍